boards/esp32-c3-devkitM-1/layout.ld

```diff
@@ -1,8 +1,8 @@
 MEMORY
 {
   rom (rx)  : ORIGIN = 0x40380000, LENGTH = 0x30000
-  ram (rwx) : ORIGIN = 0x403B0000, LENGTH = 0x30000
```

Comment:
How did this work?!?

Reply:
Based on the datasheet it appears to me as both of these sections are located in SRAM. Perhaps you made the same mistake as I did and read the `B` as an `8`?

Reply:
It's inside the SRAM memory space

Reply:
So if you unplug the board the apps go away?

Reply:
No, it's confusing. Tock and apps are loaded into flash. The boot ROM then loads that flash into RAM and executes it.

So the read-only regions of Tock need to be in RAM as that is where the boot ROM jumps to. In theory we could store the apps in flash, but the ROM seems to do all sorts of strange things so I haven't been able to get that to works. Plus I don't think that would get us much as Tock can't use fragmented memory addresses